charts/consul/templates/server-statefulset.yaml

```diff
@@ -254,6 +261,9 @@ spec:
                 {{- range $value := .Values.global.recursors }}
                 -recursor={{ quote $value }} \
                 {{- end }}
+                {{- if (and .Values.dns.enabled .Values.dns.enableRedirection)}}
+                -recursor="$kube_dns_service_ip" \
```

Comment:
this variable could contain multiple nameservers, so i think we need to iterate 😬